src/Silk.Core/Commands/Server/TagCommand.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using DSharpPlus.Interactivity.Enums;
+using DSharpPlus.Interactivity.Extensions;
+using MediatR;
+using Silk.Core.Services;
+using Silk.Core.Utilities.HelpFormatter;
+using Silk.Data.MediatR;
+using Silk.Data.Models;
+using Silk.Extensions;
+
+namespace Silk.Core.Commands.Server
+{
+    [RequireGuild]
+    [Group("tag")]
+    [Category(Categories.Server)]
+    public class TagCommand : BaseCommandModule
+    {
+        private readonly IMediator _mediator;
+        private readonly TagService _tagService;
+        private readonly string[] _reservedWords = new[]
+        {
+            "create", "update", "delete",
+            "alias", "info", "claim"
+        };
+        public TagCommand(IMediator mediator, TagService tagService)
+        {
+            _mediator = mediator;
+            _tagService = tagService;
+        }
+
+        [GroupCommand]
+        public async Task Tag(CommandContext ctx, [RemainingText] string tag)
+        {
+            Tag? dbTag = await _tagService.GetTagAsync(tag, ctx.Guild.Id);
+            
+            if (dbTag is null)
+            {
+                await ctx.RespondAsync("Tag not found!");
+            }
+            else
+            {
+                await ctx.RespondAsync(dbTag.Content);
+                await _mediator.Send(new TagRequest.Update(tag, ctx.Guild.Id) {Uses = dbTag.Uses + 1});
```

Comment:
This `Uses` count is only going to show the uses of this particular tag and not its aliases.  I'm not sure what it is intended for but one could imagine you would want the `Uses` to be shared across a tag and all of its aliases.

Reply:
Perhaps. Hadn't really thought about it, but as I said, aliases are tags, technically speaking, so it's kind of the same side of a coin depending on how you look at it.